### PR TITLE
fix: Provide more context when launcher is not available

### PR DESCRIPTION
**Pull request by MichalBryxi**
- Add information which launchers are available
- Helps with #1837
- Example of new error message:
```
Launcher IE not found. Not installed? Available: safari
```

### CODE REVIEW

**Comment by johanneswuerbach** [comment]
@MichalBryxi could you rebase your PR? This should resolve the CI issues.

**Comment by MichalBryxi** [comment]
Done @johanneswuerbach

File: tests/ci/ci_tests.js [hunk 1]
```diff
@@ -223,12 +223,15 @@ describe('ci mode app', function() {
...
     config.read(function() {
-      var app = new App(config, function(exitCode, err) {
-        expect(exitCode).to.eq(1);
-        expect(err.message).to.eq('Launcher ' + browser + ' not found. Not installed?');
-        done();
+      config.getAvailableLaunchers((err, launchers) => {
+        var app = new App(config, function(exitCode, err) {
+          expect(exitCode).to.eq(1);
+
+          expect(err.message).to.eq('Launcher ' + browser + ' not found. Not installed? Available: ' + Object.keys(launchers));
```
Comment (johanneswuerbach) [inline tests/ci/ci_tests.js:230]: Looks like there is a missing join here.

```suggestion
          expect(err.message).to.eq('Launcher ' + browser + ' not found. Not installed? Available: ' + Object.keys(launchers).join(', '));
```

Comment (MichalBryxi) [reply tests/ci/ci_tests.js]: Heh, possible test coverage improvement :) Added it to [todos](https://github.com/testem/testem/issues/1837)

**Comment by MichalBryxi** [comment]
No idea what is the CI complaining about now, but seems like the shape of the PR is ok now?

**Comment by johanneswuerbach** [comment]
Looks like Windows was just being Windows, thank you 🎉

**Comment by MichalBryxi** [comment]
Thank you good sir for letting my future self be confused about this a little bit less next time!

**Pull request merged**